modules/cnf-installing-the-operators.adoc

```diff
@@ -0,0 +1,261 @@
+
+.Procedure
+
+. Create the `operator-namespace.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    openshift.io/cluster-monitoring: "true"
+  name: openshift-performance-addon-operator
+spec: {}
+----
+
+. Create the `operator_operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: performance-addon-operator
+  namespace: openshift-performance-addon-operator
+----
+
+. Run the following command to get the `channel` value required for the next step.
++
+[source,terminal]
+----
+$ oc get packagemanifest performance-addon-operator -n openshift-marketplace -o jsonpath='{.status.defaultChannel}'
+----
++
+.Example output
+----
+4.6
+----
+
+. Create the following Subscription CR and save the YAML in the `pao-sub.yaml` file:
++
+.Example subscription
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-performance-addon-operator-subscription
+  namespace: openshift-performance-addon-operator
+spec:
+  channel: "<channel>" <1>
+  name: performance-addon-operator
+  source: redhat-operators <2>
+  sourceNamespace: openshift-marketplace
+----
+<1> Specify the value you obtained in the previous step for the `status.defaultChannel` parameter.
+<2> You must specify the `redhat-operators` value.
+
+. Configure the `kustomization.yaml` file:
++
+[source,yaml]
+----
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# TODO handle namespace with var
+
+resources:
+  - operator-namespace.yaml
+  - operator_operatorgroup.yaml
+  - operator_subscription.yaml
+----
+
+[id="cnf-installing-the-precision-time-protocol-operator_{context}"]
+== Installing the Precision Time Protocol (PTP) operator
+
+Install the PTP Operator using the {product-title} CLI or the web console.
+
+.Procedure
+
+. Create the `01_namespace.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-ptp
+  labels:
+    openshift.io/run-level: "1"
+    openshift.io/cluster-monitoring: "true"
+----
+
+. Create the `02_operator_group.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: ptp-operators
+  namespace: openshift-ptp
+spec:
+  targetNamespaces:
+    - openshift-ptp
+----
+
+. Create the `03_subscription.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ptp-operator-subscription
+  namespace: openshift-ptp
+spec:
+  channel: "${OC_VERSION}"
+  name: ptp-operator
+  source: "redhat-operators"
+  sourceNamespace: openshift-marketplace
+----
+
+. Configure the `kustomization.yaml` file:
++
+[source,yaml]
+----
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - 01_namespace.yaml
+  - 02_operator_group.yaml
+  - 03_subscription.yaml
+----
```

Comment:
```suggestion

```

Reply:
You want this entire step and codeblock removed?

Reply:
Yes

Reply:
Change made.